packages/react-components/react-radio/src/components/Radio/useRadio.tsx

```diff
@@ -58,6 +59,7 @@ export const useRadio_unstable = (props: RadioProps, ref: React.Ref<HTMLInputEle
       defaultChecked,
       disabled,
       required,
+      'aria-describedby': describedByGroup,
```

Comment:
Shouldn't you merge the one that comes from the context + the one that comes from the props?

```suggestion
      'aria-describedby': `${describedByGroup} ${props["aria-describedby"]}`,
```

Reply:
I don't think so. If someone sets `aria-describedby` on the Radio, then they are overriding the default value. They'd be responsible for including (or not including) the one from the RadioGroup.

Reply:
We also don't know enough about the two describedby strings to know if they can be concatenated, and what order to concatenate them.

Reply:
In that case, should this then be

```suggestion
      'aria-describedby': props['aria-describedby'] ?? describedByGroup,
```

Reply:
`...nativeProps.primary` is spread just below this, and will override `'aria-describedby'` if present.

Reply:
I updated ariaDescribedBy to work more like other group props (get its default value when spreading props). The result is the same, but now it works the same as the other props.

Reply:
Thanks Ben, resolving this convo.